src/routes/adminPanel/adminPanel.component.tsx

```diff
@@ -0,0 +1,240 @@
+                    updateMyData(parseInt(id), Header, value);
+                  }}
+                >
+                  {radarQuadrants?.map(({ position, name }: RadarQuadrant) => {
+                    return (
+                      <option key={name} value={position}>
+                        {name}
+                      </option>
+                    );
+                  })}
+                </StyledSelect>
+              );
+            },
+          },
+          {
+            Header: 'ring',
+            accessor: 'ring',
+            Cell: (row) => {
+              const {
+                value,
+                updateMyData,
+                row: { id },
+                column: { Header },
+              } = row;
+
+              return (
+                <StyledSelect
+                  value={value}
+                  onChange={(e) => {
+                    const {
+                      target: { value },
+                    } = e;
+                    updateMyData(parseInt(id), Header, value);
+                  }}
+                >
+                  {radarRings?.map(({ position, name }: RadarRing, index: number) => {
+                    return (
+                      <option key={name} value={index}>
+                        {name}
+                      </option>
+                    );
+                  })}
+                </StyledSelect>
+              );
+            },
+          },
+          {
+            Header: 'description',
+            accessor: 'description',
+          },
+          {
+            Header: 'specification',
+            accessor: 'specification',
+          },
+          {
+            Header: 'github',
+            accessor: 'github',
+          },
+          {
+            Header: 'projects',
+            accessor: 'projects',
+          },
+          {
+            Header: 'iconDescription',
+            accessor: 'iconDescription',
+          },
+          {
+            Header: 'iconName',
+            accessor: 'iconName',
+          },
+          {
+            Header: 'iconUrl',
+            accessor: 'iconUrl',
+            Cell: (row) => {
+              const { value } = row;
+              return <img src={value} alt="img" height={HEIGHT} />;
+            },
+          },
+          {
+            Header: 'alternatives',
+            accessor: 'alternatives',
+            Cell: (row) => {
+              const {
+                value,
+                updateMyData,
+                row: { id },
+                column: { Header },
+              } = row;
+              const options = radarTechnologies?.map((tech: RadarTechnology) => ({ value: tech.id, ...tech }));
+
+              return (
+                <InlineSelectContainer>
+                  <Select
+                    defaultValue={value}
+                    isMulti
+                    options={options}
+                    onChange={(data) => {
+                      const updatedData = data.map((item) => {
```

Comment:
please don't mutate an item. It will be better to make 
`data.map(({value, ...rest}) => (rest))`